src/test/java/com/google/api/codegen/testdata/java_main_library.baseline

```diff
@@ -233,21 +247,45 @@ public class LibraryServiceApi implements AutoCloseable {
 
   /**
    * Parses the archive_path from the given fully-qualified path which
-   * represents a archivedBook resource.
+   * represents a archived_book resource.
```

Comment:
This happens for the consistency with existing path templates -- it's "archived_book" in `formatArchivedBookName` already. Because it's in a comment, it would usually indicate something in "archived_book", not a variable name, so I think this would be okay.